src/channels/index.tsx

```diff
@@ -11,15 +11,24 @@ import {
   channelPath,
   channelsListPath,
   ChannelsListUrlQueryParams,
-  ChannelsListUrlSortField
+  ChannelsListUrlSortField,
+  ChannelUrlQueryParams
 } from "./urls";
 import ChannelCreateComponent from "./views/ChannelCreate";
 import ChannelDetailsComponent from "./views/ChannelDetails";
 import ChannelsListComponent from "./views/ChannelsList";
 
-const ChannelDetails: React.FC<RouteComponentProps<{ id: string }>> = ({
-  match
-}) => <ChannelDetailsComponent id={decodeURIComponent(match.params.id)} />;
+const ChannelDetails: React.FC<RouteComponentProps<any>> = ({ match }) => {
+  const qs = parseQs(location.search.substr(1));
+  const params: ChannelUrlQueryParams = qs;
```

Comment:
This variable assignment doesn't make sense, you're referencing the object above.